### PR TITLE
fix slow file type detection in all data import

### PR DESCRIPTION
**Pull request by robinschmid**
By default we were using in input stream two times to determine the correct mzML import: IMS or no IMS
Now we are handling IMS and mzML files the same way with the advanced MSDK import.

This actually took a long time sometimes for ~500 files.

Also, I am checking if the file name ends with .mzml case insensitive.

### CODE REVIEW

File: src/main/java/io/github/mzmine/util/RawDataFileTypeDetector.java [hunk 1]
```diff
@@ -99,31 +94,38 @@ public static RawDataFileType detectDataFileType(File fileName) {
...
+              return RawDataFileType.AIRD;
+            }
           }
+          logger.info("It's not an aird format file or the aird index file not exist");
+        }
+        if (fileName.getName().contains(TDF_SUFFIX) || fileName.getName()
+            .contains(TDF_BIN_SUFFIX)) {
+          return RawDataFileType.BRUKER_TDF;
+        }
+        if (fileName.getName().contains(TDF_SUFFIX) || fileName.getName()
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/util/RawDataFileTypeDetector.java:120]: TSF

File: src/main/java/io/github/mzmine/util/RawDataFileTypeDetector.java [hunk 2]
```diff
@@ -160,7 +162,7 @@ public static RawDataFileType detectDataFileType(File fileName) {
         }
 
         if (fileHeader.contains(MZML_HEADER)) {
-          return getMzmlType(fileName);
+          return RawDataFileType.MZML;
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/util/RawDataFileTypeDetector.java:165]: I think im missing it, but where is the discrimination between ims and non ims?

File: src/main/java/io/github/mzmine/util/RawDataFileTypeDetector.java (hunk 1 above)
Comment (robinschmid) [reply src/main/java/io/github/mzmine/util/RawDataFileTypeDetector.java:120]: fixed. must have been like this for a while?

File: src/main/java/io/github/mzmine/util/RawDataFileTypeDetector.java (hunk 2 above)
Comment (robinschmid) [reply src/main/java/io/github/mzmine/util/RawDataFileTypeDetector.java:165]: no discrimination needed at this point as we are doing this already in the mzml import task

**Comment by robinschmid** [comment]
ready from my side

File: src/main/java/io/github/mzmine/util/RawDataFileTypeDetector.java (hunk 1 above)
Comment (SteffenHeu) [reply src/main/java/io/github/mzmine/util/RawDataFileTypeDetector.java]: yeah

**Pull request merged**